### DISCUSSION TITLE
MorphToMany has no method relatableQueryUsing()

### DISCUSSION THREAD

**Discussion by msucevan** [laravel/nova-issues#6494] [Ideas & Feature Requests] [answered]
### Description:
As documented the [MorphToMany](https://nova.laravel.com/docs/resources/relationships.html#searchable-relations) field is searchable, but the method relatableQueryUsing() does not exist.

### Detailed steps to reproduce the issue on a fresh Nova installation:
```php
\Laravel\Nova\Fields\MorphToMany::make(__('Factors'), 'factors', FactorResource::class)
            ->searchable()
            ->showCreateRelationButton()
            ->relatableQueryUsing(function (NovaRequest $request, Builder $query) {
                return $query->withoutOriginal();
            })
```

**Comment by marcobax** [comment]
I stumbled upon this today while I was working on creating a MorphToMany relation field. Since it's not supported by default, did you manage to find a solution for this? The **relatableQuery** and **reletable{MyModel}** doesn't seem to be triggered.

**Comment by crynobone** [comment, marked as answer]
Implemented in 5.1.0
